diff --git a/node_modules/uni-axios-ts/dist/index.d.ts b/node_modules/uni-axios-ts/dist/index.d.ts
index 1d44958..8343b6c 100644
--- a/node_modules/uni-axios-ts/dist/index.d.ts
+++ b/node_modules/uni-axios-ts/dist/index.d.ts
@@ -35,7 +35,7 @@ interface ImplicitlyAny {
 /**
  * Axios 请求参数
  */
-interface AxiosRequestConfig {
+export interface AxiosRequestConfig {
     [key: string]: any
     baseURL?: string
     url?: string
